README.md

```diff
@@ -20,7 +20,36 @@ This instance is not backed up, and will expire 90 days from creation. Export an
 
 IMPORTANT: Reboots of the VM are not supported, and will result in loss of data. Back up any datasets or models prior to a reboot or shutdown of underlying VPC infrastructure.
 
-NOTE: Please note that provisioning may take approximately fifteen minutes.
+NOTE: Please note that provisioning may take approximately twenty minutes.
+
+
+## Standalone Terraform Deployment Steps
+
+### Prerequisites
+
+To run as a standalone Terraform deployment, you need the following prerequisites.
+
+```
+terraform: v0.11.x
+ibm terraform provider: v0.22.x
```

Comment:
The versions here are actually backdated - but its nbd. it'll be fine with this. We'll need to update the version when we get the resource-group stuff in.